### PR TITLE
Allow disabling markdown check steps in the default build test

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: scripts/presubmit-tests.sh [hunk 1]
```diff
@@ -103,21 +107,33 @@ function run_build_tests() {
...
     check_links_in_markdown ${mdfiles} || failed=1
   fi
+  return ${failed}
+}
+
+# Default build test runner that:
+# * check markdown files
+# * `go build` on the entire repo
+# * run `/hack/verify-codegen.sh` (if it exists)
+# * check licenses in `/cmd` (if it exists)
```
Comment (chaodaiG) [inline scripts/presubmit-tests.sh]: this line is not accurate any more. not part of this PR, so this can be done in a separate PR

**Comment by chaodaiG** [comment]
overall lgtm, with a minor comment

File: scripts/presubmit-tests.sh (hunk 1 above)
Comment (adrcunha) [reply scripts/presubmit-tests.sh]: Technically this is a nit on an outdated comment.

Comment (adrcunha) [reply scripts/presubmit-tests.sh]: Updated here and in the README.

**Comment by chaodaiG** [comment]
> > overall lgtm, with a minor comment
> 
> Such comment doesn't help productivity. I'm aware of the comment, because you published it. You did not "/lgtm", so I can't merge the PR. So, for the sake of saving your time and mine, and being more productive, you can either a) request the change, b) add the comment, then /lgtm and /hold so I can decide if I'm gonna address the comment here or in a future PR, merging this one (by unholding it).

I see, good to know

**Pull request merged**